public/scripts/bioController.js

```diff
@@ -0,0 +1,55 @@
+(function (module) {
+  var bio = {};
+
+  var buildBio = function (proto) { // Takes a legislator object and creates a new legislator object with specific properties.
+    var bioInfo = {};
+    if (proto){
```

Comment:
Are there other properties available on the `proto` argument object? If not, why not just use an iteration tool like `forEach` to run `verify()`?